internal/security/proxy/service.go

```diff
@@ -173,8 +173,12 @@ func (s *Service) postCert(cert CertificateLoader) error {
 		if err != nil {
 			return err
 		}
-		e := fmt.Sprintf("failed to add certificate with errorcode %d, error %s", resp.StatusCode, string(b))
+		reason := string(b)
+		e := fmt.Sprintf("failed to add certificate with errorcode %d, error %s", resp.StatusCode, reason)
 		LoggingClient.Error(e)
+		if (resp.StatusCode == http.StatusBadRequest) && (strings.Index(reason, "existing certificate") != -1) {
+			e = fmt.Sprintf("certificate already exists on reverse proxy")
```

Comment:
@tingyuz why the http status code not be `http.StatusConflict` when the certificate already exists?

Reply:
FWIW the docs don't seem to say what happens when there's already a cert uploaded: https://docs.konghq.com/1.0.x/admin-api/#create-certificate

Reply:
Please refer my comments on #2089. The original code even checks http.StatusConflict but kong doesn't follow such standard, so we have to get this done this way.

Reply:
ok